### DISCUSSION TITLE
Pen stroke thickness is different from freehand to straight line in Wayland

### DISCUSSION THREAD

**Discussion by mstorti** [xournalpp/xournalpp#4570] [General] [answered]
Hi guys, I find  Xournal++ very helpful, thank you for that!

Recently I migrated from Xorg to Wayland (Fedora 37) and it seems that several things changed. The strokes for the pen in free hand mode are much smooth which is good . 

But on the negative side I see that the thicknes o the strokes are much thinner now for the freehand mode. In the  picture below, "Hello" is written in freehand mode, and the box with the shape mode,

![image](https://user-images.githubusercontent.com/61880905/211221960-25a9fd88-180b-44df-ae68-1d1cd97d7524.png)

As a workaround I 'm changing the thickness of the stroke each time I change from freehand to straight line, but it is very tiring. 

TIA

**Comment by rolandlo** [comment, marked as answer]
@mstorti Nice to hear that you like our app.
Please try out the pressure multiplier setting in the Stylus tab of menu `Edit > Preferences`. You seem to need a larger value there. This should fix your problem. Make sure you have the latest version of Xournal++ (maybe even the nightly build).

**Reply by mstorti** [reply]
Thank you very much rolandlo! That was the fix to my problem!
